README.md

```diff
@@ -1,47 +1,56 @@
 # Mantle: Gluing CoreOS together
 
-This repository is a collection of utilities for the CoreOS SDK.
-
-## plume
-
-CoreOS release utility
-
-## plume index
-
-Generate and upload index.html objects to turn a Google Cloud Storage
-bucket into a publicly browsable file tree. Useful if you want something
-like Apache's directory index for your software download repository.
+This repository is a collection of utilities for the Container Linux SDK. Most of the
```

Comment:
Maybe "for developing Container Linux"? I'm all for fixing it now since most of the original readme was out of date/no longer accurate. 

I'm not familair with gangue. Where all is it used (so I can see how it's used and write a description that makes sense)?